### PR TITLE
Git basics updates 

### PR DESCRIPTION
**Pull request by Lynnmwova**
I have contributed some information that you might find useful to the repository. Kindly review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,17 @@ It covers:
   -   Add files to be tracked by Git `git add 'my-file'`.
   -   Stage changes to be commited `git add 'my-file'`.
   -   Commit changes with appropriate commit message `git commit -m 'commit message'`.
+  - Finding out information regarding what files are modified and those that are in the staging area using `git status`.
```
Comment (Peter-Nyarunda) [inline README.md]: My opinion is of this line being between tracking and committing a file. Kindly look at it and provide your thoughts .

File: README.md [hunk 2]
```diff
@@ -8,3 +8,17 @@ It covers:
...
+  - Printing out all the commits that are completed using `git log`.
```
Comment (Peter-Nyarunda) [inline README.md]: This line reflects on additional information on working with Git hence not particularly on the trunk workflow but rather as a N/B or point to note, what do you think?

File: README.md [hunk 3]
```diff
@@ -8,3 +8,17 @@ It covers:
...
+  - Pushing all code from local repository to the remote repository on GitHub using `git push -u origin master`
+  -  Using `git pull origin master` to pull the latest changes from the remote repository to local
```
Comment (Peter-Nyarunda) [inline README.md]: This Line and the next appear to be contained in a particular step and should have it's own topic, thoughts?

File: README.md [hunk 4]
```diff
@@ -8,3 +8,17 @@ It covers:
   -   Add files to be tracked by Git `git add 'my-file'`.
   -   Stage changes to be commited `git add 'my-file'`.
   -   Commit changes with appropriate commit message `git commit -m 'commit message'`.
+  - Finding out information regarding what files are modified and those that are in the staging area using `git status`.
+  - Printing out all the commits that are completed using `git log`.
+  - Pushing all code from local repository to the remote repository on GitHub using `git push -u origin master`
```
Comment (Peter-Nyarunda) [inline README.md]: Kindly provide a subtopic illustrating the process through which one sets up and uses remotes.

File: README.md [hunk 5]
```diff
@@ -8,3 +8,17 @@ It covers:
...
+  -  Using `git pull origin master` to pull the latest changes from the remote repository to local
+  - Cloning existing repositories to your PC using `git clone [repository url]`.
+  
+
+
+# _A brief introduction to Git_
```
Comment (Peter-Nyarunda) [inline README.md]: This section is great. This should be our starter section, give this a slightly higher level heading and give it a bit of more content, brief and straight to the point

File: README.md [hunk 6]
```diff
@@ -8,3 +8,17 @@ It covers:
...
+---
+Real life projects often require multiple devs working in parallel. A version control system (Git) is useful in handling code updates, their history and changes. Git also enables branching and merging.
+
+What is a branch? _A branch is a pointer to the latest commit in the git repository_ . Multiple branches are required to support parallel project development. Create a new branch using the command `git branch myBranch`. This command creates a branch called myBranch. In order to switch from the main branch to myBranch, use `git checkout myBranch`. In order to list all the branches in local, the `git branch` command comes in handy.
```
Comment (Peter-Nyarunda) [inline README.md:43]: Branching is a another workflow and requires its own subtopic , what are your thoughts on separating branching to a subtopic and offering more instructions on setting and working with branches.

**Comment by Peter-Nyarunda** [comment]
Greeting Lynnmwova,
I have gone through your requested changes and have attached my thoughts on areas to edit, 
kindly review them and provide your thoughts and changes if need there be.
Great ideas though on introducing a small introduction on Git as a VCS, and branching.
Looking forward to hearing your response.
Regards
Peter-Nyarunda

File: README.md (hunk 1 above)
Comment (Lynnmwova) [reply README.md]: That sounds like a suitable positioning for it. Change to be implemented in the next PR

File: README.md (hunk 2 above)
Comment (Lynnmwova) [reply README.md]: It is a helpful trick to know just in case.

File: README.md (hunk 3 above)
Comment (Lynnmwova) [reply README.md]: I think I should introduce a new topic that deals with working with remotes and synchronizing changes to both remote and local.

File: README.md (hunk 4 above)
Comment (Lynnmwova) [reply README.md]: On it

File: README.md (hunk 5 above)
Comment (Lynnmwova) [reply README.md]: Moving the introduction paragraph to the top of the readme

File: README.md (hunk 6 above)
Comment (Lynnmwova) [reply README.md:43]: A branching topic will be implemented once the basics are covered.

File: README.md (hunk 3 above)
Comment (Peter-Nyarunda) [reply README.md]: That would work best

**Review by Peter-Nyarunda — APPROVED** [review]

**Pull request merged**